modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ClusterPerClassIntegrationTest.java

```diff
@@ -515,14 +513,20 @@ public static void waitForIndexBuild(String tableName, String indexName) throws
                     continue;
                 }
 
+                TablesView tablesView = getTablesConfiguration(clusterNode).value();
+
                 IndexStorage index = internalTable.storage().getOrCreateIndex(
                         partitionId,
-                        createIndexDescription(getTablesConfiguration(clusterNode).value(), indexName)
+                        createIndexDescriptor(tablesView, tablesView.indexes().get(indexName.toUpperCase()).id())
```

Comment:
please extract `tablesView.indexes().get(indexName.toUpperCase()).id()` into a variable

Reply:
fix it